src/dom-to-image-more.js

```diff
@@ -236,27 +235,41 @@
     }
 
     function cloneNode(node, filter, root, parentComputedStyles, ownerWindow) {
-        if (!root && filter && !filter(node)) {
+        if (!root && (filter && !filter(node))) {
```

Comment:
I might have fixed part of this over in #104 by making `cloneNode` skip any node with a tagName of  `'SCRIPT'`...

Reply:
Best to merge #104 to master and then deal with the line in the merge conflict.

Reply:
No longer in the PR diff! :)